mod/wet4/views/default/page/elements/head.php

```diff
@@ -231,7 +231,7 @@
 preg_match("/groups\/profile\/[\d]*\/.*\/?/", $_SERVER['REQUEST_URI'], $output_array);
 if (sizeof($output_array) > 0) {
   if ($my_page_entity instanceof ElggGroup && $my_page_entity->getContentAccessMode() !== "unrestricted") {
```

Comment:
Do we need this if now?

Reply:
we may need it in the future.. but for now the client wants users to be able to look up all the groups, even if it is restricted to group members only.

i will remove it for now